OpenRA.Mods.Common/Traits/Upgrades/DeployToUpgrade.cs

```diff
@@ -32,6 +34,18 @@ public class DeployToUpgradeInfo : ITraitInfo, Requires<UpgradeManagerInfo>
 		[Desc("Cursor to display when unable to (un)deploy the actor.")]
 		public readonly string DeployBlockedCursor = "deploy-blocked";
 
+		[Desc("Facing that the actor must face before deploying. Will deploy disregarding facing by default.")]
+		public readonly int? Facing = null;
```

Comment:
IMO it would be better to keep this consistent with the rest of our code.  Somebody could argue for changing all of them in a different pr, but the other parts of the code may run into performance regressions due to unboxing.